remote/registry/regclient/client_test.go

```diff
@@ -48,7 +48,7 @@ func NewTestRunner(t *testing.T) (*TestRunner, func()) {
 		AcceptSizeMax: -1,
 		Enabled:       true,
 		AllowRemoves:  true,
-	}, r.Logbook())
+	}, r.Logbook(), node.Repo.Bus())
```

Comment:
qq: what's the difference between `registry/...` and `remote/registry/...` since they are near identical copies. I focused my efforts on the `registry/...` piece not sure if/why I would bring that over.